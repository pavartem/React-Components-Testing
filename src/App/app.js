import React, { Component } from "react";
import Posts from "../Posts/posts";
import Pagination from "../Pagination/pagination";

class App extends Component {

  render() {
    return (
      <>
        <h1>App</h1>
        <Posts/>
        <Pagination/>
      </>
    );
  }
}

export default App;
