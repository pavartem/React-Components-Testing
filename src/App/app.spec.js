import React from "react";
import App from "./app";

const setUp = () => shallow(<App />);

describe("Info component", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });

  it("should render Info component", () => {
    expect(component).toMatchSnapshot();
  });
});
