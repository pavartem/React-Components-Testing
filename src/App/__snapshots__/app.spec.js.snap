// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info component should render Info component 1`] = `
<Fragment>
  <h1>
    App
  </h1>
  <Posts />
  <Pagination
    lastPage={0}
    onClick={[Function]}
    page={0}
  />
</Fragment>
`;
